Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/50cal.yml

```
@@ -50,28 +50,28 @@
 
-# .50 Cal AP
+# 57mm Cal AP
 
 - type: entity
-  id: 50BulletArmorPiercing
-  name: .50 AP bullet
-  parent: 50BulletBase
+  id: 57mmBulletArmorPiercing
+  name: 57mm AP bullet
+  parent: 57mmBulletBase
   categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)